admin-ui-deployment.yml

```diff
@@ -109,17 +73,6 @@ properties:
     url: (( param "specify uaa url" ))
     admin:
       client_secret: (( param "specify uaa client secret" ))
-  tripwire: (( param "specify tripwire properties" ))
-  awslogs: (( param "specify awslogs properties" ))
-  nessus-agent: (( param "specify nessus properties" ))
-  newrelic: (( param "specify newrelic properties" ))
-  collectd:
-    riemann_server: (( param "specify riemann host" ))
-    hostname_prefix: (( param "specify collectd prefix" ))
-
-  riemann:
```

Comment:
Hmm, 🤔 did the runtime config get changed? I didn't notice `riemann` coming through earlier when I was deploying Shibb 🐕.